src/Maestro/SubscriptionActorService/PullRequestDescriptionBuilder.cs

```diff
@@ -176,28 +176,30 @@ public int GetStartingReferenceId()
         return regex.Matches(_description.ToString()).Select(m => Int32.Parse(m.ToString())).DefaultIfEmpty(0).Max() + 1;
     }
 
-    public static string GetChangesURI(string repoURI, string from, string to)
+    public static string GetChangesURI(string repoURI, string fromSha, string toSha)
     {
         if (repoURI == null)
         {
             throw new ArgumentNullException(nameof(repoURI));
         }
-        if (from == null)
+        if (fromSha == null)
         {
-            throw new ArgumentNullException(nameof(from));
+            throw new ArgumentNullException(nameof(fromSha));
```

Comment:
Btw there is now this:
https://learn.microsoft.com/en-us/dotnet/api/system.argumentnullexception.throwifnull?view=net-8.0

```csharp
ArgumentNullException.ThrowIfNull(fromSha);
```

Reply:
Ah, that's nice, thanks